packages/device_info/device_info/example/lib/main.dart

```diff
@@ -59,13 +59,13 @@ class _MyAppState extends State<MyApp> {
 
   Map<String, dynamic> _readAndroidBuildData(AndroidDeviceInfo build) {
     return <String, dynamic>{
-      'version.securityPatch': build.version.securityPatch,
-      'version.sdkInt': build.version.sdkInt,
-      'version.release': build.version.release,
-      'version.previewSdkInt': build.version.previewSdkInt,
-      'version.incremental': build.version.incremental,
-      'version.codename': build.version.codename,
-      'version.baseOS': build.version.baseOS,
+      'version.securityPatch': build.version?.securityPatch,
+      'version.sdkInt': build.version?.sdkInt,
+      'version.release': build.version?.release,
+      'version.previewSdkInt': build.version?.previewSdkInt,
+      'version.incremental': build.version?.incremental,
+      'version.codename': build.version?.codename,
+      'version.baseOS': build.version?.baseOS,
```

Comment:
should `version` be non-nullable?  In what situation should it be `null`?

Reply:
All these values in `AndroidDeviceInfo` came from method channel as a map, so I feel like it is probably safest to make them nullable? As we don't have control over the result from method channel. 
The other way I can think of is to provide a default value to some of the variables so they can be non-nullable. I'm not sure if that's a better approach tho.